source/overview/corba.rst

```diff
@@ -0,0 +1,32 @@
+10 things you should know about CORBA
+=====================================
+
+1.  ***You don’t need to know CORBA to work with TANGO***
```

Comment:
Please remove additional asterixes (two on both sides of text will do the work).